lib/rock/webapp/tasks/api.rb

```diff
@@ -8,6 +9,12 @@ class API < Grape::API
                 version 'v1', using: :header, vendor: :rock
                 format :json
 
+                if ENV["REST_API_LOG_CONSOLE"]
+                    puts "logging"
```

Comment:
You forgot the first part ... it would IMO make sense to use the logger to log the message instead of sending to the console indiscriminately.